src/mame/konami/konppc_jvshost.h

```diff
@@ -0,0 +1,40 @@
+// license:BSD-3-Clause
+// copyright-holders:windyfairy
+#ifndef MAME_KONAMI_KONPPC_JVSHOST_H
+#define MAME_KONAMI_KONPPC_JVSHOST_H
+
+#pragma once
+
+#include "emu.h"
+
```

Comment:
Please don’t `#include "emu.h"` from headers, only as the first thing in .cpp files – it causes issues with prefix header voodoo otherwise.